experimenter/experimenter/jetstream/tasks.py

```diff
@@ -2,11 +2,12 @@
 
 import markus
 from celery.utils.log import get_task_logger
+from django.core.cache import cache
```

Comment:
So, by default this will use the [in memory cache](https://docs.djangoproject.com/en/4.2/topics/cache/#local-memory-caching) which means this data won't be available in the request handlers.  So we'd either have to change the cache backend to point to redis, which we do have deployed and available

https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/settings.py#L343-L345

but isn't currently [configured](https://docs.djangoproject.com/en/4.2/topics/cache/#redis) so if you wanna use that you'd have to either land that in here or as a separate PR I think?

Reply:
Ah great! This was something I was wondering, so good to know. This part is easy to change, is there a better solution to make the data available for the front end? I think you mentioned something last week but I forget what it was and cannot find any notes on the discussion.